otterdog/eclipse-keyple.jsonnet

```diff
@@ -403,6 +403,54 @@ orgs.newOrg('eclipse-keyple') {
         orgs.newEnvironment('github-pages'),
       ],
     },
+    orgs.newRepo('keyple-less-distributed-client-kmp-lib') {
+      allow_merge_commit: true,
+      allow_update_branch: false,
+      delete_branch_on_merge: false,
+      description: "Eclipse Keyple™ Project: Kotlin Multiplatform distributed remote client lib for Keyple-less terminals",
+      gh_pages_build_type: "legacy",
+      gh_pages_source_branch: "gh-pages",
+      gh_pages_source_path: "/",
+      homepage: "https://keyple.org/",
+      secret_scanning: "disabled",
```

Comment:
I would enable secret scanning and push protection for all repos, we usually enable it for all repos.